addons/html_builder/static/src/plugins/image/image_tool_option_plugin.js

```diff
@@ -142,6 +142,64 @@ class ImageToolOptionPlugin extends Plugin {
                     editingElement.replaceWith(newImage);
                 },
             },
+            setLink: {
+                apply: ({ editingElement }) => {
+                    const parentEl = this.searchSupportedParentLinkEl(editingElement);
+                    if (parentEl.tagName !== "A") {
+                        const wrapperEl = document.createElement("a");
+                        editingElement.after(wrapperEl);
+                        wrapperEl.appendChild(editingElement);
+                    } else {
+                        const fragment = document.createDocumentFragment();
+                        fragment.append(...parentEl.childNodes);
+                        parentEl.replaceWith(fragment);
+                    }
+                },
+                isApplied: ({ editingElement }) => {
+                    const parentEl = this.searchSupportedParentLinkEl(editingElement);
+                    return parentEl.tagName === "A";
+                },
+            },
+            setUrl: {
+                apply: ({ editingElement, value }) => {
```

Comment:
You can add preview:"false" (You action is not previewable). You can remove preview="false" from your template. (It s new in the api)